stylesheet.css

```diff
@@ -450,3 +450,124 @@ header {
 .skill {
   margin-bottom: 2vh;
 }
+
+.contact_me {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  max-width: 100vw;
+  min-height: 100vh;
+  padding-bottom: 10vh;
+}
+
+.contact_me_intro {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  max-width: 85vw;
+  color: #172b40;
+  padding-top: 10vh;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 2rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 2.75rem;
+}
+
+fieldset {
+  border: none;
+}
+
+.personal_info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
```

Comment:
- [ ] Great job on implementing the contact form as specified in the project requirement! 👏 However, there are some minor UI issues 🐞 that we need to address and fix before moving on to the next milestone. To do that, kindly consider the below suggestions:
  - [ ] Kindly consider adhering to the UI design as shown on the `Figma` design template as it going to help you create a professional-looking portfolio and also create a good user experience. The styles of your `input` fields doesn't match that of the `Figma` design so consider styling your project to match it. See below for more details:
  
  <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="353" alt="Screenshot 2023-07-20 at 14 36 19" src="https://github.com/ommarz/Portfolio/assets/107192076/5c453d74-f2c4-4166-8e44-d9dbea5cdb90">
</td>
<td>
<img width="374" alt="Screenshot 2023-07-20 at 14 34 53" src="https://github.com/ommarz/Portfolio/assets/107192076/7aa9ee33-8638-4864-bd74-a9724feefcaf">
</td>
</tr>
</table>